de_DE/webapp/src/main/webapp/templates/freemarker/visualization/capabilitymap/capabilityMap_de_DE.ftl

```diff
@@ -15,8 +15,13 @@ ${stylesheets.add(
 )}
 
 <script language="JavaScript" type="text/javascript">
+    var i18nStringsCap = {
+        term: '${i18n().group_capitalized}',
```

Comment:
These appear to be swapped: `term` is being populated by "group_capitalized".

Reply:
You are right, i changed it